gap/SkeletalFinSetsForCAP.gi

```diff
@@ -909,14 +911,14 @@ AddClassifyingMorphismOfSubobjectWithGivenSubobjectClassifier( SkeletalFinSets,
     
     range := Range( monomorphism );
     
-    images := List( [ 1 .. Length( range ) ],
+    images := List( [ 0 .. Length( range ) - 1 ],
                     function ( x )
                       
                       if x in AsList( monomorphism ) then
                           return 1;
                       fi;
                       
-                      return 2;
```

Comment:
Changing the roles of 1 and 2 (resp. 0 and 1 in the new setup) is an intended change, correct?

Reply:
This is intended: 1 was the value of true and should remain the value of true.